src/app/team/service/details/ServiceDetailsContainer.jsx

```diff
@@ -31,6 +31,16 @@ class ServiceDetailsContainer extends Component {
     return `/team/location/${locationId}/services/${serviceId}`;
   };
 
+  filteredServiceFields = () => (
+    SERVICE_FIELDS.filter((el) => {
```

Comment:
I just noticed 1 other bug: This does filter out the irrelenvat fields from the _list_, but when a field is done and the next one needs to be chosen, there's no filtering.
So, for instance, I can do the following:
1. Go into a Food service
2. Click into the "PLHIV" field
3. Enter a value, click "OK"
4. Click "Yes" to confirm

What I expect is to go to "Membership", which is the next field on the list. But what actually happens is that I get to "Occasions", which doesn't even exist for Food services. (Same thing happens if I just use the down/up arrows to navigate between the fields.)

Reply:
Yes, you are right, i just fixed this